presto-main/src/test/java/com/facebook/presto/tz/TestJdkBasedZoneInfoProvider.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.tz;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.format.DateTimeFormat;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class TestJdkBasedZoneInfoProvider
```

Comment:
There are quite a few methods on Joda's DateTimeZone class.  I am concerned that we may have missed or in the future they may change, so that our overrides are not sufficient.  Can we get more coverage of these extra methods?

Reply:
I implemented all the abstract methods, and abstract methods only. I didn't override anything.